.github/workflows/lint.yml

```diff
@@ -0,0 +1,52 @@
+name: lint
+
+on:
+  push:
+    branches: [develop, main]
+  pull_request:
+    branches:
+      - "**"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+env:
+  CARGO_TERM_COLOR: always
+  BINSTALL_NO_CONFIRM: true
+
+jobs:
+  clippy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/actions/rust
+      - run: cargo clippy --all-targets -- -D warnings -A incomplete-features
+      - run: cargo clippy --package zero --all-targets --no-default-features --features cdk_erigon -- -D warnings -A incomplete-features
```

Comment:
nit: copy-pasta, but the 2nd job should be applied to the workspace level, not just `zero` package